include/libtorrent/torrent.hpp

```diff
@@ -1610,7 +1603,13 @@ namespace libtorrent
 		// the number of bytes of padding files
 		boost::uint32_t m_padding:24;
 
-		// TODO: gap of 8 bits available here
+		// this is set to the connect boost quota for this torrent.
+		// After having received this many priority peer connection attempts, it
+		// fallse back onto the steady state peer connection logic, driven by the
+		// session tick.  The first tracker response, when this is non-zero will
+		// attempt to connect to a bunch of peers immediately and set this to
+		// false. We only do this once to get the torrent kick-started
```

Comment:
This comment needs (more) updating, the variable is not longer immediately set to false.